### PR TITLE
Computed the averages correctly in even based damage

### PR DESCRIPTION
**Pull request by micheles**
Closes https://github.com/gem/oq-engine/issues/5802.

### CODE REVIEW

File: openquake/calculators/scenario_damage.py [hunk 1]
```diff
@@ -200,11 +200,19 @@ def post_execute(self, result):
         events_per_asset = (indices[:, 1] - indices[:, 0]).mean()
         logging.info('Found ~%d dmg distributions per asset', events_per_asset)
 
+        # avg_factor = factor used when computing the averages
+        oq = self.oqparam
+        if oq.investigation_time is None:  # scenario_damage
```
Comment (raoanirudh) [inline openquake/calculators/scenario_damage.py:205]: If the job file for a scenario_damage calculation includes the `investigation_time` inadvertently, do we ignore it when reading the oq params?

Comment (micheles) [reply openquake/calculators/scenario_damage.py:205]: I would raise an error in that case, what do you think?

Comment (raoanirudh) [reply openquake/calculators/scenario_damage.py:205]: That should work

> I would raise an error in that case, what do you think?

That should work

Comment (micheles) [reply openquake/calculators/scenario_damage.py:204]: Done

**Pull request merged**